xml/security_ldap_active_directory_sync.xml

```diff
@@ -0,0 +1,157 @@
+      plans for both &ds389a; and &ada; to ensure that you correctly restore 
+      only a single replication connection between these topologies.
+    </para>
+    <screen>
+┌────────────────┐       ┌────────────────┐               ┌────────────────┐       ┌────────────────┐
+│                │       │                │               │                │       │                │
+│     389-ds     │◀─────▶│     389-ds     │◀ ─ ─ ─ ─ ─ ─ ▶│       AD       │◀─────▶│       AD       │
+│                │       │                │               │                │       │                │
+└────────────────┘       └────────────────┘               └────────────────┘       └────────────────┘
+         ▲                        ▲                                ▲                        ▲        
+         │                        │                                │                        │        
+         ▼                        ▼                                ▼                        ▼        
+┌────────────────┐       ┌────────────────┐               ┌────────────────┐       ┌────────────────┐
+│                │       │                │               │                │       │                │
+│     389-ds     │◀─────▶│     389-ds     │               │       AD       │◀─────▶│       AD       │
+│                │       │                │               │                │       │                │
+└────────────────┘       └────────────────┘               └────────────────┘       └────────────────┘
+</screen>
+</sect2>
+
+<sect2 xml:id="sec-security-ldap-synchronization-ad-prerequisites">
+  <title>Prerequisites for &ad;</title>
+  <para>
+    A security group that is granted the "Replicating Directory Changes" 
+    permission is required. For example, you have created a group named 
+    "Directory Server Sync". Follow the steps in the "How to grant the 
+    'Replicating Directory Changes' permission for the Microsoft Metadirectory Services ADMA service account" 
+    (<link xlink:href="https://docs.microsoft.com/en-US/troubleshoot/windows-server/
+    windows-security/grant-replicating-directory-changes-permission-adma-service"/> to set this up.
+  </para>
+  <warning>
+    <title>Strong security needed</title>
+    <para>
+      You should consider members of this group to be of equivalent security 
+      importance to Domain Administrators. Members of this group have the 
+      ability to read sensitive content from the Active Directory 
+      environment, so you should use strong, randomly-generated service 
+      account passwords for these accounts, and carefully audit membership to this group.
+    </para>
+  </warning>
+  <para>
+    You should also create a service account that is a member of this group. 
+  </para>
+  <para>
+    Your &ad; environment must have certificates configured for LDAPS to 
+    ensure that authentication between &ds389a; and &ada; is secure. The 
```

Comment:
"Authentication with GSSAPI ..." not "The gssap ..."

Reply:
:+1: